<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1058</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1041</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Module 1</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>321</width>
          <height>451</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Serial port: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="serialPorts"/>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>RX:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="currentRXData"/>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>TX:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txData"/>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Mini terminal:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="miniterminal"/>
         </item>
         <item>
          <widget class="QPushButton" name="connectBtn">
           <property name="text">
            <string>Connect to device</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Module 2</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>321</width>
          <height>451</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Serial port: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="serialPorts_2"/>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>RX:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="currentRXData_2"/>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>TX:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txData_2"/>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Mini terminal:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout"/>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="miniterminal_2"/>
         </item>
         <item>
          <widget class="QPushButton" name="connectBtn_2">
           <property name="text">
            <string>Connect to device</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Module 3</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>321</width>
          <height>451</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Serial port: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="serialPorts_3"/>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>RX:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="currentRXData_3"/>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>TX:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txData_3"/>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Mini terminal:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="miniterminal_3"/>
         </item>
         <item>
          <widget class="QPushButton" name="connectBtn_3">
           <property name="text">
            <string>Connect to device</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1058</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
